Library/Formula/websockify.rb

```diff
@@ -0,0 +1,70 @@
+require "FileUtils"
+
+class Websockify < Formula
+  homepage "https://github.com/kanaka/websockify"
+  url "https://github.com/kanaka/websockify/archive/v0.6.0.tar.gz"
+  sha1 "f2bffebc9e4d94ad3a329638bee63e899eaad9c0"
+
+  depends_on :python if MacOS.version <= :snow_leopard
```

Comment:
websockify has a numpy dependency. System python comes with numpy but a user-installed python may not.

Do we want to make this
```ruby
if MacOS.version <= :snow_leopard
  depends_on :python
  depends_on "numpy" => :python
end
```

or do something more elaborate?